apps_wishlist.md

```diff
@@ -106,10 +97,8 @@ The following list is a compiled wishlist of applications that would be nice-to-
 - [LocomotiveCMS](https://github.com/locomotivecms/engine)
 - [Logstalgia](http://logstalgia.io)
 - [Loomio](https://www.loomio.org)
-- [MagicMirror²](https://github.com/MichMich/MagicMirror)
 - [MaidSafe](http://maidsafe.net)
 - [Mailpile](https://www.mailpile.is)
-- [Mailtrain](https://mailtrain.org/)
```

Comment:
You sure about removing those ? (Sorry maybe you already answered that question)

Reply:
No, I'm correcting that.

Reply:
https://github.com/YunoHost/doc/pull/1095/commits/16d5221f0ca78fcf2110db1feba9c9d9b909902a